### PR TITLE
[CA-1431] merge dev

### PR DESCRIPTION
**Pull request by s-rubenstein**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -10201,6 +10431,50 @@ components:
...
+          description: the name of the service permeters for google project
+        invalidBillingAccount:
+          type: boolean
+          description: whether or not the billing account is usable by Terra
+        roles:
+          type: array
+          items:
+            type: string
+          description: the roles the caller has on the project
+        workspacesWithCorrectBillingAccount:
```
Comment (s-rubenstein) [inline src/main/resources/swagger/api-docs.yaml:10458]: Adding these two fields and `WorkspaceBillingAccount` are the only two manual changes I made, everything else was from the merge command.

**Review by marctalbott — APPROVED** [review]

**Comment by s-rubenstein** [comment]
Jenkins retest

**Review by gpolumbo-broad — APPROVED** [review]

**Comment by s-rubenstein** [comment]
Jenkins retest

**Comment by s-rubenstein** [comment]
Jenkins retest

**Comment by s-rubenstein** [comment]
Jenkins retest

**Comment by s-rubenstein** [comment]
Here's a passing test run with my firecloud develop branch: https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/52259/

**Pull request merged**